eosdis-nasa/earthdata-pub-api
# EDPUB-1332: Service Authorizer Updates

# Description

Various updates to get service authorizer in working condition for ORNL testing

## Spec

Designs: https://docs.google.com/document/d/1XYQ3eZJixL4sRqoGrdVWl92B95u7xN3MRexdVySREgM/edit#heading=h.cka31ha7pojn

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1332

---

## Validation

1. Make sure all merge request checks have passed (CI/CD)
2. Pull related branches locally
3. Register new service to database with accessible endpoint
4. Register new service id to `ornl_service_trigger` step
5. Initialize new request and reassign to `ornl_test_workflow`
6. Obtain submission secret from registered endpoint
7. Use encoded service id + submission secret to perform action with one of the service endpoints.

---
